models/issues/issue.go

```diff
@@ -109,12 +133,13 @@ type Issue struct {
 	Milestone        *Milestone             `xorm:"-"`
 	Project          *project_model.Project `xorm:"-"`
 	Priority         int
-	AssigneeID       int64            `xorm:"-"`
-	Assignee         *user_model.User `xorm:"-"`
-	IsClosed         bool             `xorm:"INDEX"`
-	IsRead           bool             `xorm:"-"`
-	IsPull           bool             `xorm:"INDEX"` // Indicates whether is a pull request or not.
-	PullRequest      *PullRequest     `xorm:"-"`
+	AssigneeID       int64             `xorm:"-"`
+	Assignee         *user_model.User  `xorm:"-"`
+	IsClosed         bool              `xorm:"INDEX"`
+	ClosedStatus     IssueClosedStatus `xorm:"INDEX NOT NULL DEFAULT 0"`
```

Comment:
Is there a query like "SELECT .. WHERE closed_status = ... " without other indexed columns? If no, this "index" only affects performance but doesn't improve.

Reply:
Do we need to add a filter of "close status" in the issue list page?

Reply:
Does the index really work even if there is a list page?